test.txt

```diff
@@ -0,0 +1 @@
+This is a plain text file for testing https://github.com/woocommerce/woocommerce-blocks/pull/6769.
```

Comment:
Looks good! Don't forget to delete this file before merging :D

Reply:
I was removing it the same time you mentioned it. 😁